RUMM-454 Update docs - outline manual view tracking mode as default mode (#269)
DataDog/dd-sdk-android: docs/rum_collection.md

  **Note**: For `ActivityViewTrackingStrategy`, `FragmentViewTrackingStrategy`, or `MixedViewTrackingStrategy` you can validate which `Fragment` or `Activity` is tracked as a RUM view event by providing a `ComponentPredicate` implementation in the constructor.

>   By default RUM Monitor for View tracking runs in manual mode so if you decide not to provide a view tracking strategy you will have to manually send us the

Reviewer: ```suggestion
  By default RUM Monitor for View tracking runs in manual mode so if you decide not to provide a view tracking strategy you will have to manually send the
```

>   views by calling the start/stop View API methods yourself.

Reviewer 2: Maybe instead of having this as a Note, you should add an example of calling `GlobalRum.get().startView()` and `GlobalRum.get().stopView()` between points 3 and 4 as 
`4. (Optionnal)`

    This creates RUM resource data around each request processed by the OkHttpClient, with all the relevant information automatically filled (URL, method, status code, error).

    **Note**: If you use multiple Interceptors, this one must be called first.

> 4. If you want to manually track your view you can call the `RumMonitor#startView` when the view becomes visible and interactive (equivalent with the lifecycle event `onResume`) followed by `RumMonitor#stopView` when the view is no longer visible(equivalent with the lifecycle event `onPause`) as follows:

Reviewer: ```suggestion
4. If you want to manually track your view, call the `RumMonitor#startView` when the view becomes visible and interactive (equivalent with the lifecycle event `onResume`) followed by `RumMonitor#stopView` when the view is no longer visible(equivalent with the lifecycle event `onPause`) as follows:
```

   ```kotlin
      fun onResume(){
        GlobalRum.get().startView(viewKey, viewName, viewAttributes)        
      }
      ...
      fun onPause(){
        GlobalRum.get().stopView(viewKey, viewAttributes)        
      }
   ```

> 6. (Optionnal) If you want to get timing information in Resources (such as time to first byte, DNS resolution, …), you can add the provided [Event][6] listener as follows:

Reviewer: ```suggestion
6. (Optional) If you want to get timing information in Resources (such as time to first byte, DNS resolution, etc.), you can add the provided [Event][6] listener as follows:
```
